course6/toyapp_alexs/.vscode/tasks.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Don't put this into git. This is your private config that reflects your private, local environment.
Add .vscode to your _private_ gitignore file!

Reply:
will be removed